src/Storefront/Resources/views/storefront/component/captcha/honeypot.html.twig

```diff
@@ -1,5 +1,5 @@
 {% block component_captcha_honeypot %}
-    <div class="captcha--honeypot">
+    <div class="register-shopware-surname">
```

Comment:
This is almost the content of the constant which is already used below for the inputs name - would you mind using it here as well? Something along the line of `constant('[...]::CAPTCHA_REQUEST_PARAMETER')|replace({'_': '-'})` should work I think.